dist/amazon-linux/2/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM amazonlinux:2 AS build
+
+ENV REFRESHED_AT 20220926T210646Z
+ENV LANG=C.UTF-8
+ENV MIX_ENV=prod
+
+# Set the right versions
+ENV ERLANG_VERSION 25.1.1
+ENV ELIXIR_VERSION 1.14.1
+
+RUN yum install -y ruby-devel git openssl-devel ncurses-devel wget && \
+    yum groupinstall -y  "Development tools" --setopt=group_package_types=mandatory && \
```

Comment:
`--setopt=group_package_types=mandatory` makes it so that we only install mandatory packages. This saves a ton of disk usage